### PR TITLE
Add last update timestamp stuff to api docs

### PR DESCRIPTION
**Pull request by dpeng817**
Last update fetchers to api docs

eyes on api docs

### CODE REVIEW

**Comment by dpeng817** [comment]
* **#20779** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/20779?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @dpeng817 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by johannkm — APPROVED** [review]
good to double check the rendering before landing

File: docs/sphinx/sections/api/apidocs/libraries/dagster-gcp.rst [hunk 1]
```diff
@@ -41,6 +41,12 @@ BigQuery Ops
 
 .. autofunction:: import_gcs_paths_to_bq
 
+
+Data Quality
```
Comment (sryza) [inline docs/sphinx/sections/api/apidocs/libraries/dagster-gcp.rst:45]: We should call this Data Freshness. In our messaging, we distinguish between the two.

**Review by sryza — CHANGES_REQUESTED** [review]

**Review by sryza — COMMENTED** [review]
Btw are these top-level APIs?

**Comment by dpeng817** [comment]
@sryza yes

File: docs/sphinx/sections/api/apidocs/libraries/dagster-gcp.rst (hunk 1 above)
Comment (dpeng817) [reply docs/sphinx/sections/api/apidocs/libraries/dagster-gcp.rst:45]: ack.

**Review by sryza — APPROVED** [review]

**Pull request merged**